tasks/remove-defaults.yml

```diff
@@ -6,6 +6,13 @@
   notify:
     - reload nginx
 
+- name: Disable the default site (on newer nginx versions)
+  file:
+    path: "{{nginx_conf_dir}}/sites-enabled/default.conf"
+    state: absent
+  notify:
```

Comment:
There are a problem if someone setup default in nginx_sites.
You can use something like that:
```   
when: >
     'default' not in nginx_sites.keys()
```

Reply:
Done, do you think that's ok?